examples/documentation_examples/examples_test.go

```diff
@@ -37,18 +37,19 @@ func TestDocumentationExamples(t *testing.T) {
 
 	db := client.Database("documentation_examples")
 
-	documentation_examples.InsertExamples(t, db)
-	documentation_examples.QueryToplevelFieldsExamples(t, db)
-	documentation_examples.QueryEmbeddedDocumentsExamples(t, db)
-	documentation_examples.QueryArraysExamples(t, db)
-	documentation_examples.QueryArrayEmbeddedDocumentsExamples(t, db)
-	documentation_examples.QueryNullMissingFieldsExamples(t, db)
-	documentation_examples.ProjectionExamples(t, db)
-	documentation_examples.UpdateExamples(t, db)
-	documentation_examples.DeleteExamples(t, db)
-	documentation_examples.RunCommandExamples(t, db)
-	documentation_examples.IndexExamples(t, db)
-	documentation_examples.StableAPIExamples()
+	stw := newSubtestWrapper(t, db)
+	stw.run("InsertExamples", documentation_examples.InsertExamples)
+	stw.run("QueryToplevelFieldsExamples", documentation_examples.QueryToplevelFieldsExamples)
+	stw.run("QueryEmbeddedDocumentsExamples", documentation_examples.QueryEmbeddedDocumentsExamples)
+	stw.run("QueryArraysExamples", documentation_examples.QueryArraysExamples)
+	stw.run("QueryArrayEmbeddedDocumentsExamples", documentation_examples.QueryArrayEmbeddedDocumentsExamples)
+	stw.run("QueryNullMissingFieldssExamples", documentation_examples.QueryNullMissingFieldsExamples)
+	stw.run("ProjectionExamples", documentation_examples.ProjectionExamples)
+	stw.run("UpdateExamples", documentation_examples.UpdateExamples)
+	stw.run("DeleteExamples", documentation_examples.DeleteExamples)
+	stw.run("RunCommandExamples", documentation_examples.RunCommandExamples)
+	stw.run("IndexExamples", documentation_examples.IndexExamples)
+	stw.runEmpty("StableAPExamples", documentation_examples.StableAPIExamples)
 
 	// Because it uses RunCommand with an apiVersion, the strict count example can only be
 	// run on 5.0+ without auth. It also cannot be run on 6.0+ since the count command was
```

Comment:
You _could_ also use an `mtest.Testing` instance for the `StableAPIStrictCountExample` like `mtest.Options().MinServerVersion("5.0").MaxServerVersion("6.0").Auth(true)` instead of the odd `testutil` and `os.Getenv` logic I wrote below.